svelte-kit-scss/src/lib/components/Profile/UserProfile/UserProfile.spec.ts

```diff
@@ -20,15 +20,20 @@ describe('UserProfile', () => {
     ['name', mockUserProfile.name],
     ['login', mockUserProfile.login],
     ['bio', mockUserProfile.bio],
-    // ['followers', mockUserProfile.followers],
-    // ['following', mockUserProfile.following],
-    // ['email', mockUserProfile.email],
-    // ['blog', mockUserProfile.blog],
-    // ['twitterUsername', mockUserProfile.twitterUsername],
-    // ['location', mockUserProfile.location],
+    ['followers', mockUserProfile.followers],
```

Comment:
Once again, imo, these should be separate tests (if the validation of the different fields would matter)